cmd.js

```diff
@@ -20,7 +21,16 @@ function cmd () {
     argv.splice(stacksOnlyIx + 1, 1)
   }
 
-  var ix = argv.indexOf('node')
+  // get the position of node binary in argv; else stay -1
+  let ix = -1
+  let bin = ''
```

Comment:
since 0.12 can emit perf events, I'm reluctant to use syntax that isn't supported in 0.12 (even though I've not tested in 0.12) 

so can we make let, var 

we can leave const, since 0.12 does support that (even though it's not the es6 const, but the old v8 const)